generator.sh

```diff
@@ -48,12 +48,27 @@ fi
 hostname=""
 bundle_identifier=""
 
+bundle_regex="^[a-zA-Z]+(\.?[a-zA-Z]+[0-9a-zA-Z\_]*)+$"
+
 while [ -z "$hostname" ]; do
     read -p'--> On Android, which host would you like to use for deep linking? (eg. www.mobify.com) ' hostname
 done
 
 while [ -z "$bundle_identifier" ]; do
     read -p"--> Which iOS Bundle Identifier and Android Package Name would you like to use? Begin with 'com.mobify.' to use HockeyApp. (eg. com.mobify.app) " bundle_identifier
+
+    if [[ "$bundle_identifier" =~ $bundle_regex ]]; then
+        read -p"--> Continue with the iOS Bundle Identifier and Android Package Name $bundle_identifier? (y/n) " -n 1 -r
+        echo
+        if [[ ! $REPLY =~ ^[Yy]$ ]] ; then
+            bundle_identifier=''
+        fi
+    else
+        bundle_identifier=''
+        echo '--> Invalid package name'
```

Comment:
I think for messages relating to an original question (like "Which bundle ID... ?") we should not use the `-->` prefix but indent the messages so that they're easy to follow. 

This is what I get when I enter an invalid bundle ID
![screen shot 2016-05-10 at 12 50 39 pm](https://cloud.githubusercontent.com/assets/77138/15160332/06f4ecca-16ae-11e6-93a1-237fe391cd01.png)

Maybe something like this? 
```
--> We have a license you must read and agree to. Read license? (y/n) y
--> I have read, understand, and accept the terms and conditions stated in the license above. (y/n) y
--> What is the name of your project? zzztest
--> Continue with the project name 'zzztest'? (y/n) y
--> On Android, which host would you like to use for deep linking? (eg. www.mobify.com)
--> On Android, which host would you like to use for deep linking? (eg. www.mobify.com) com.mobi-fy28(*($*!(*~(*$.test
--> Which iOS Bundle Identifier and Android Package Name would you like to use? Begin with 'com.mobify.' to use HockeyApp. (eg. com.mobify.app) com.mobi-fy92498(#@*$%)(@*#%_()*~!)(#*)(*#@)(*%.test
     Invalid package name: com.mobi-fy92498(#@*$%)(@*#%_()*~!)(#*)(*#@)(*%.test
     The name may contain uppercase or lowercase letters (A through Z), numbers, and underscores (_).
     However, individual package name parts may only start with letters.
--> Which iOS Bundle Identifier and Android Package Name would you like to use? Begin with 'com.mobify.' to use HockeyApp. (eg. com.mobify.app)
```